src/Tools/MicroBuild/post-build.ps1

```diff
@@ -0,0 +1,79 @@
+# Run our post build steps.  This occurs after all of our shipping binaries are built and fully 
+# signed.  The steps we need to execute are:
+#
+#   1. Run GitLink on the PDBs so we have proper source linking.
+#   2. Setup the <Configuration>\Index directory for simple / efficient indexing in official builds
+
+
+param([string]$config = "Release")
+Set-StrictMode -version 2.0
+$ErrorActionPreference="Stop"
+
+# Get the path to all of the shipping EXE / DLL files.
+function Get-BinaryList() {
+    $list = @()
+    $config = Join-Path $repoDir "build\config\SignToolData.json"
+    $j = ConvertFrom-Json (Get-Content -raw $config)
+    foreach ($entry in $j.sign) {
+        foreach ($v in $entry.values) { 
+            $ext = [IO.Path]::GetExtension($v)
+            if (($ext -eq ".dll") -or ($ext -eq ".exe")) {
+                $path = Join-Path $configDir $v
+                $list += $path;
+            }
+        }
+    }
+
+    return $list
+}
+
+# Run GitLink on all of our PDB files in their original location.  
+function Run-GitLink($binaryLst) {
+
+    $gitlinkVersion = Get-PackageVersion "GitLink"
+    $gitlink = Join-Path (Get-PackagesDir) "GitLink\$gitlinkVersion\build\GitLink.exe"
+
+    Write-Host "Running GitLink"
+    foreach ($filePath in $binaryList) { 
+        $pdbPath = [IO.Path]::ChangeExtension($filePath, ".pdb")
+        Write-Host "`t$pdbPath"
+        Exec-Block { & $gitlink -u "https://github.com/dotnet/roslyn" $pdbPath --baseDir $repoDir } | Write-Host
+    }
+}
+
+# Create single directory to hold all of the items that we need to be indexed by 
```

Comment:
Is the intent of this to limit the indexing jobs we're shipping to the corporate symbol server? What impact does this also make for symbol archiving?

I guess I'm not exactly sure what "official build system" and "the indexing script" means here. My first read thought it meant *this* script which was really confusing.